packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -60,6 +60,18 @@ class CodePushClientWrapper {
 
   final CodePushClient codePushClient;
 
+  Future<List<AppMetadata>> getApps() async {
+    final fetchAppsProgress = logger.progress('Fetching apps');
+    try {
+      final apps = (await codePushClient.getApps()).toList();
```

Comment:
Very nit, feel free to ignore: this might be ever so slightly cleaner if the `.toList` was below on the `return` line, which would allow you to not have the parens around the codePushClient call.

Reply:
will update in the next PR 👍